code/common/apidetails.q

```diff
@@ -13,6 +13,8 @@ add[`.api.find;1b;"Generic method for finding functions/variables/tables/views.
 add[`.api.search;1b;"Generic method for searching all function definitions for a specific string. s is based on this";"[string: search string; boolean: whether the search is context senstive";"table of matching functions and definitions"]
 add[`.api.add;1b;"Add a function to the api description table";"[symbol:the name of the function; boolean:whether it should be called externally; string:the description; dict or string:the parameters for the function;string: what the function returns]";"null"]
 add[`.api.fullapi;1b;"Return the full function api table";"[]";"api table"]
+add[`.api.exportconfig;1b;"Return value table of requested torq variables and descriptions";"[result of .api.f]";"keyed table of name, value, description"]
```

Comment:
The input is the list of namespaces to return isn't it? 
e.g. .api.exportconfig[`.proc`.servers] 
? 
The description is a bit confusing

Reply:
That's correct, it's the namespace. Will change